libs/flags/yaml_flag.go

```diff
@@ -0,0 +1,37 @@
+package flags
+
+import (
+	"fmt"
+	"os"
+	"sigs.k8s.io/yaml"
+)
+
+type YamlFlag struct {
+	raw []byte
+}
+
+func (y *YamlFlag) String() string {
+	return fmt.Sprintf("YAML (%d bytes)", len(y.raw))
+}
+
+// TODO: Command.MarkFlagFilename()
```

Comment:
Can you propagate this? The override already exists, so you could mark the flags there.

Reply:
Nevermind, I just saw this was copy/pasted from the json flag.